### DISCUSSION TITLE
Cannot access to the GitHub development package

### DISCUSSION THREAD

**Discussion by hengyu23333** [community/community#54417] [GitHub Education]
### Hi everyone,

I last verified as a student on May 2, 2022. I'd like to know why I don't need to reverify now. There may be a waiting period between verification and access to academic benefits. I don't know how long should I wait for?

**Comment by PPKk99** [comment]
You have to be patient and research, there are many solutions, you just have to learn from all experiences

**Comment by samus-aran** [comment]
Hey @hengyu23333  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
